tests/unit/components/UI/Text/URL.spec.js

```diff
@@ -0,0 +1,81 @@
+import { mount } from "@vue/test-utils";
+import URL from "@/components/UI/Text/URL";
+import Tooltip from "primevue/tooltip";
+
+// mock document.execCommand
+document.execCommand = jest.fn();
+
+describe("URL.vue", () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(URL, {
+      global: {
+        directives: {
+          tooltip: Tooltip,
+        },
+      },
+    });
+  });
+
+  it("should render with default values", () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("link renders correctly", async () => {
+    const link = "testLink";
+    await wrapper.setProps({ link: link });
+
+    expect(wrapper.get('[data-test="link"]').text()).toBe(link);
+  });
+
+  it("link text class is set correctly", async () => {
+    const link = "testLink";
+    const linkClass = "w-12 h-12";
+    await wrapper.setProps({
+      link: link,
+      urlStyleClass: linkClass,
+    });
+
+    expect(wrapper.get('[data-test="link"]').classes()).toEqual(
+      expect.arrayContaining(["w-12", "h-12"])
```

Comment:
Isn't it better to cross check this with the variable "linkClass" that you created? Just trying to avoid hardcoding